aws/lambda/oss_ci_cur/lambda_function.py

```diff
@@ -92,10 +93,13 @@ def get_clickhouse_client(
     host: str, user: str, password: str
 ) -> clickhouse_connect.driver.client.Client:
     # for local testing only, disable SSL verification
-    return clickhouse_connect.get_client(host=host, user=user, password=password, secure=True, verify=False)
+    # return clickhouse_connect.get_client( host=host, user=user, password=password, secure=True, verify=False )
```

Comment:
I don't know our lintrunners (nor how to write for TypeScript), but space after parethesis...